src/diffcalc_API/routes/Constraints.py

```diff
@@ -15,6 +15,15 @@
 singleConstraintType = Union[Tuple[str, float], str]
 
 
+@router.get("/{name}")
+async def get_constraints_status(
+    name: str, hklCalc: HklCalculation = Depends(unpickleHkl)
+):
+    return Response(
+        content=hklCalc.constraints.__str__(), media_type="application/text"
```

Comment:
Is it appropriate to return a string? Would we be better of serialising to `JSON`?

Reply:
Ah, so the reason this is returning a string is because I want the terminal to print a lovely long message with newlines when this route is called. And I discovered that if it was inside a JSON object, this would not be the case. Effectively, the response content looks something like:

'<<crystal orientations>>

h     k      l       tag
0     0     1       orient1
0     0     0.5    orient2

<< UB Matrix >>
....
'

and so on.